divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x,z):
```

Comment:
I think there is a tpyo here and it should be...

```suggestion
def divide(x,y):
```

Or you could `return x/z`